test/integration/patients/patient/sidebar.js

```diff
@@ -821,6 +821,120 @@ context('patient sidebar', function() {
       .should('not.contain', 'Workspace Two');
   });
 
+  specify('workspace specific widgets setting', function() {
+    const dob = testDateSubtract(10, 'years');
+
+    const fields = {
+      'test-field': {
+        id: '1',
+        name: 'test-field',
+        value: '1',
+      },
+    };
+
+    cy
+      .routesForPatientDashboard()
+      .routeFormDefinition()
+      .routeFormFields()
+      .routeForm()
+      .routeSettings(fx => {
+        fx.data[0].attributes = {
+          value: {
+            widgets: [
+              'dob',
+              'divider',
+              'workspaces',
+              'divider',
+              'optionsWidget1',
+            ],
+            fields: [
+              'test-field',
+            ],
+          },
+        };
+
+        return fx;
+      })
+      .routeWidgets(fx => {
+        const addWidget = _.partial(getResource, _, 'widgets');
+
+        fx.data = fx.data.concat([
+          addWidget({
+            id: 'optionsWidget1',
+            widget_type: 'optionsWidget',
+            definition: {
+              display_name: 'Populated Option Widget',
+              field_name: 'test-field',
+              display_options: {
+                '1': 'Test Field',
+              },
+            },
+          }),
+        ]);
+
+        return fx;
+      })
+      .routePatient(fx => {
+        fx.data.id = '1';
+        fx.data.attributes = {
+          first_name: 'First',
+          last_name: 'Last',
+          birth_date: dob,
+        };
+
+        fx.data.relationships['patient-fields'].data = [{
+          id: '1',
+          type: 'patient-fields',
+        }];
+
+        return fx;
+      })
+      .routeWorkspaces(fx => {
+        fx.data[0].attributes.settings = {
+          widgets_patient_sidebar: {
+            widgets: [
+              'dob',
+              'optionsWidget1',
+            ],
+            fields: [
+              'test-field',
+            ],
+          },
+        };
+
+        return fx;
+      });
+
+    _.each(_.keys(fields), fieldName => {
+      cy.routePatientField(fx => {
+        fx.data = getResource(fields['test-field'], 'patient-fields');
+        return fx;
+      }, 'test-field');
+    });
+
+    cy
+      .visit('/patient/dashboard/1')
+      .wait('@routePatient')
+      .wait('@routePrograms')
+      .wait('@routeWidgets');
+
+    _.each(_.keys(fields), fieldName => {
+      cy.wait(`@routePatientField${ fieldName }`);
+    });
```

Comment:
I don't think you need to test a patient field here at all.  Just add the simplest widget.. for that matter it could just be a divider.  Just needs to prove it overrides the default,

Reply:
My assumption was that we needed to cover the [getSidebarWidgets()](https://github.com/RoundingWell/care-ops-frontend/blob/develop/src/js/services/bootstrap.js#L104-L106) pulling from the workspace setting instead of the org setting.

But that would make the test a lot simpler without the field.

Reply:
I think we can rely on the default of `routeSettings` to set the sidebar we're testing to overwrite.